### PR TITLE
APEXMALHAR-2434 Use fixed/settable metaQueueName in JMSTransactionableStore. This closes #612.

### PR DESCRIPTION
**Pull request by sanjaypujare**
@PramodSSImmaneni pls review and merge.

### CODE REVIEW

**Comment by pramodin** [comment]
@sanjaypujare since the majority of the contribution was done by @oliverwnk, would you be ok with changing commit author to him with your name as a contributing author within the commit message. Also, looks like there are checkstyle errors failing the build.

**Comment by sanjaypujare** [comment]
@PramodSSImmaneni sure I am okay with the commit author suggestion. Will fix the checkstyle issues

**Comment by pramodin** [comment]
Minor suggestion, for your name in commit message use git style syntax name <email> instead of github style.

Thanks

File: library/src/main/java/com/datatorrent/lib/io/jms/JMSTransactionableStore.java [hunk 1]
```diff
@@ -58,24 +65,38 @@ public JMSTransactionableStore()
   {
   }
 
+  /**
+   * @return the metaQueueName
```
Comment (pramodin) [inline library/src/main/java/com/datatorrent/lib/io/jms/JMSTransactionableStore.java]: Description of the method

**Pull request merged**